go.sum

```diff
@@ -26,7 +26,6 @@ github.com/kr/pty v1.1.1 h1:VkoXIwSboBpnk99O/KFauAEILuNHv5DVFKZMBN/gUgw=
 github.com/kr/pty v1.1.1/go.mod h1:pFQYn66WHrOpPYNljwOMqo10TkYh1fy3cYio2l3bCsQ=
 github.com/kr/text v0.1.0 h1:45sCR5RtlFHMR4UwH9sdQ5TC8v0qDQCHnXt+kaKSTVE=
 github.com/kr/text v0.1.0/go.mod h1:4Jbv+DJW3UT/LiOwJeYQe1efqtUx/iVham/4vfdArNI=
-github.com/myitcv/gopherjs v0.0.0-20181106113007-59ba804b0d30 h1:ZGxBik0evtvAinQthsLvCU0nQyYcK1tv+d0KoyIitGo=
```

Comment:
`go mod tidy` removed this, and I have no idea why. I'm on `go version devel +fe2c588b1c Sat Nov 10 13:39:01 2018 +0000 linux/amd64`.